src/extern/meson.build

```diff
@@ -0,0 +1 @@
+subdir('c-terminal')
```

Comment:
You probably want to use https://mesonbuild.com/Subprojects.html instead.

Incidentally, Meson will automatically clone git submodules for you, if it needs to in order to satisfy a meson subproject.

Reply:
Yeah i thought of that but for a quick conversion this was a simple solution. I will convert to a supproject soon